tests/model/issue.model.spec.ts

```diff
@@ -1,143 +1,57 @@
-import { IssueDispute } from '../../src/app/core/models/issue-dispute.model';
+import * as moment from 'moment';
 import { Issue } from '../../src/app/core/models/issue.model';
-import { Phase } from '../../src/app/core/models/phase.model';
-import { TesterResponse } from '../../src/app/core/models/tester-response.model';
-import { TEAM_4 } from '../constants/data.constants';
-
+import { Milestone } from '../../src/app/core/models/milestone.model';
 import { ISSUE_WITH_ASSIGNEES, ISSUE_WITH_EMPTY_DESCRIPTION } from '../constants/githubissue.constants';
 
 describe('Issue model class', () => {
   describe('.createPhaseBugReportIssue(githubIssue)', () => {
     it('correctly creates a bug reporting issue that has an empty description', async () => {
       const issue = Issue.createPhaseBugReportingIssue(ISSUE_WITH_EMPTY_DESCRIPTION);
-      expect(issue.title).toBe('App starts to lag when given large amount of input');
-      expect(issue.description).toBe('No details provided by bug reporter.');
-      expect(issue.severity).toBe('Medium');
-      expect(issue.type).toBe('FunctionalityBug');
-    });
-  });
-  describe('.createPhaseTeamResponseIssue(githubIssue, githubComment)', () => {
-    it('correctly creates a team response issue that has an empty team response', async () => {
-      const dummyTeam = null;
-      const issue = Issue.createPhaseTeamResponseIssue(ISSUE_WITH_ASSIGNEES, dummyTeam);
-      expect(issue.title).toBe('Screen freezes');
-      expect(issue.teamResponse).toBe('No details provided by team.');
-      expect(issue.severity).toBe('Medium');
-      expect(issue.assignees).toContain('anubh-v');
-      expect(issue.assignees.length).toBe(1);
+      expect(issue.globalId).toEqual(ISSUE_WITH_EMPTY_DESCRIPTION.id);
+      expect(issue.id).toEqual(ISSUE_WITH_EMPTY_DESCRIPTION.number);
+      expect(issue.created_at).toEqual(moment(ISSUE_WITH_EMPTY_DESCRIPTION.created_at).format('lll'));
+      expect(issue.title).toEqual(ISSUE_WITH_EMPTY_DESCRIPTION.title);
+      expect(issue.description).toEqual('No details provided by bug reporter.');
+      expect(issue.updated_at).toEqual(moment(ISSUE_WITH_EMPTY_DESCRIPTION.updated_at).format('lll'));
+      expect(issue.closed_at).toEqual(moment(ISSUE_WITH_EMPTY_DESCRIPTION.closed_at).format('lll'));
+      expect(issue.milestone).toEqual(
+        ISSUE_WITH_EMPTY_DESCRIPTION.milestone ? new Milestone(ISSUE_WITH_EMPTY_DESCRIPTION.milestone) : Milestone.DefaultMilestone
```

Comment:
Sorry didn't catch this earlier but this seems to be the implementation of the method? The expected result should be a new `Milestone` right? It doesn't really make sense to put logic in the expected test result.

And then you can create another test spec with a `GithubIssue` without a milestone to check if the milestone is `Milestone.DefaultMilestone`.